apps/ledger-live-mobile/src/hooks/storageProvider/useStorage.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { MobileStorageProvider } from "./MobileStorageProvider";
+
+const StorageContext = React.createContext(new MobileStorageProvider());
+
+export const useStorage = () => {
```

Comment:
Same with [this comment](https://github.com/LedgerHQ/ledger-live/pull/7572/files#r1721954788)